libraries/aws_auto_scaling_group.rb

```diff
@@ -34,11 +34,15 @@ def initialize(opts = {})
       @tags = []
 
       auto_scaling_group[:tags].map { |tag|
-        @tags.push({ 'resource_id':         tag.resource_id,
```

Comment:
don't commit this file along with this PR.. I guess we already merged this change